st2api/st2api/controllers/v1/actionalias.py

```diff
@@ -106,6 +109,28 @@ def match(self, action_alias_match_api, **kwargs):
             pecan.abort(http_client.BAD_REQUEST, str(e))
             return [self._match_tuple_to_dict(match) for match in e.matches]
 
+    @request_user_has_permission(permission_type=PermissionType.ACTION_ALIAS_HELP)
+    @jsexpose(arg_types=[str, str, int], body_cls=ActionAliasHelpAPI,
+    status_code=http_client.ACCEPTED)
```

Comment:
```
10.2.3 202 Accepted

The request has been accepted for processing, but the processing has not been completed. The request might or might not eventually be acted upon, as it might be disallowed when processing actually takes place. There is no facility for re-sending a status code from an asynchronous operation such as this.

The 202 response is intentionally non-committal. Its purpose is to allow a server to accept a request for some other process (perhaps a batch-oriented process that is only run once per day) without requiring that the user agent's connection to the server persist until the process is completed. The entity returned with this response SHOULD include an indication of the request's current status and either a pointer to a status monitor or some estimate of when the user can expect the request to be fulfilled.
```

Status code 202 should only be used in cases when server can not answer within 30 second response window. In this case,  `200 OK` should be used instead.